### PR TITLE
Changed knockout main to be the non minified version for debug purposes

### PR DESCRIPTION
**Pull request by carlosagsmendes**
The file should be minified during the build process

### CODE REVIEW

File: package-overrides/github/knockout/knockout@3.3.0.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-  "main": "dist/knockout",
+  "main": "dist/knockout.debug",
   "shim": {
     "dist/knockout": {
```
Comment (guybedford) [inline package-overrides/github/knockout/knockout@3.3.0.json]: This will also need to be updated to the new path.

**Comment by carlosagsmendes** [comment]
Is there an easy way to point to my local version of the registry to test things out before sending the PRs (and give you all this extra work)?

**Comment by guybedford** [comment]
Yes you can follow the steps at https://github.com/jspm/jspm-cli/wiki/Endpoints#creating-a-private-registry.

**Comment by guybedford** [comment]
Thanks.

**Pull request merged**